data/strings/types_strings.txt

```diff
@@ -3359,27 +3384,34 @@
 
   [type.boundary.administrative.city]
     en = boundary-administrative-city
+    pl = Granica miasta
 
   [type.boundary.administrative.country]
     en = boundary-administrative-country
+    pl = Granica państwa
 
   [type.boundary.administrative.county]
     en = boundary-administrative-county
+    pl = Granica państwa
```

Comment:
Are you sure?
https://wiki.openstreetmap.org/wiki/Tag:place%3Dcounty

Reply:
You're right. Should be "pl = Hrabstwo", like it is described on Polish version of mentioned wiki site. Thanks for control!

Reply:
"pl = Granica hrabstwa", of course.

Reply:
County boundary already fixed!